src/api/handler.ts

```diff
@@ -60,25 +65,51 @@ const context = async ({ event, context }) => {
       Camera: new AuthedCameraModel(user),
       MLModel: new AuthedMLModelModel(user),
       Batch: new AuthedBatchModel(user),
-      BatchError: new AuthedBatchErrorModel(user)
-    }
+      BatchError: new AuthedBatchErrorModel(user),
+    },
   };
 };
 
 const apolloserver = new ApolloServer({
-  includeStacktraceInErrorResponses: process.env.STAGE === 'dev',
+  includeStacktraceInErrorResponses: OFFLINE_MODE || process.env.STAGE === 'dev',
   status400ForVariableCoercionErrors: true,
   typeDefs,
   resolvers,
-  csrfPrevention: true,
-  cache: 'bounded'
+  csrfPrevention: false,
```

Comment:
hmm curious why we're turning this off.

Reply:
There was some cruft that I mistakenly left in this PR from trying to get local development to work. Removed in [d2efca9](https://github.com/tnc-ca-geo/animl-api/pull/189/commits/d2efca993fd1107776272d06ff5c643435878bcb)